src/stactools/modis/stac.py

```diff
@@ -103,6 +105,8 @@ def create_item(
         create_cogs (bool): Should we create cogs from the source data? If so, put
             them in `cog_directory`, or if that is `None`, put them alongside the
             hdf file.
+        raster_footprint (bool): Create the Item geometry from the convex
+            hull of valid raster data?
```

Comment:
Probably should add a note that this argument requires `create_cogs`.

Reply:
Done.